packages/mui-material/src/Autocomplete/autocompleteClasses.ts

```diff
@@ -18,6 +18,8 @@ export interface AutocompleteClasses {
   tagSizeSmall: string;
   /** Styles applied to the tag elements, for example the chips if `size="medium"`. */
   tagSizeMedium: string;
+  /** Styles applied when the autocomplete supports multiple selections. */
```

Comment:
```suggestion
  /** Styles applied to the Input when the autocomplete supports multiple selections. */
```
Because it is only applied to the input element.